scripts/install

```diff
@@ -35,6 +35,7 @@ app=$YNH_APP_INSTANCE_NAME
 
 admin_mail=$(ynh_user_get_info --username=$admin --key="mail")
 admin_pass=$(ynh_string_random --length=24)
+secrets_peertube=$(ynh_string_random --length=24)
```

Comment:
I have a slight doubt for this line (the configuration file rather suggests using an `openssl rand` command).

Looking at the code, I think that will be probably be fine.

Still, I submitted my question here: https://framacolibri.org/t/possible-value-for-secrets-peertube-in-production-yaml/16644

Reply:
OK, that seems all good!